From 70cd612f7da99449ba0c9bbbbc0ef1b3bef43f19 Mon Sep 17 00:00:00 2001
From: Simone Caronni <negativo17@gmail.com>
Date: Thu, 5 Sep 2024 09:48:57 +0200
Subject: [PATCH] Make the build noarch again.

With the autogeneration of the pkg-config file, depending on the
architecture you build the package, the file gets installed in one of
the architecure dependent folders (/usr/lib64/pkgconfig,
/usr/lib/pkgconfig, /usr/lib/x86_64-linux-gnu/pkgconfig/, etc.). This
means it would require one build for architecture.

Make it noarch again (it's just headers) by forcing the installation of
the pkg-config file in /usr/share/pkgconfig.
---
 meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 57a52af..adbce84 100644
--- a/meson.build
+++ b/meson.build
@@ -4,7 +4,8 @@ project('eglexternalplatform',
 pkg = import('pkgconfig')
 pkg.generate(filebase: 'eglexternalplatform',
   name: 'EGL External Platform interface',
-  description: 'EGL External Platform interface')
+  description: 'EGL External Platform interface',
+  install_dir: get_option('datadir') / 'pkgconfig')
 
 install_headers(
   'interface/eglexternalplatform.h',
